org.radare.Cutter.yml

```diff
@@ -0,0 +1,51 @@
+id: org.radare.Cutter
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: Cutter
+rename-desktop-file: Cutter.desktop
+rename-icon: cutter
+rename-appdata-file: Cutter.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+
+modules:
+  - name: radare
+    # translation of the arch build script
+    config-opts:
+      - --with-syszip
+      - --with-openssl
+    sources:
+      - type: archive
+        url: https://github.com/radare/radare2/archive/2.6.0.tar.gz
+        sha256: c4722519e53bbb03289e3b98e89e637e2d1f401d02cf78a198e42c2e92aeb820
+      - type: patch
+        path: fix-capstone.patch
+      - type: patch
+        path: fix-zip.patch
+      - type: patch
+        path: fix-lib-dist.patch
+      - type: git
+        url:  https://github.com/aquynh/capstone.git
+        commit: 37569a6874c8547b349a80823adda9284499fe80
+        dest: shlr/capstone
+
+  - name: cutter
+    buildsystem: qmake
+    subdir: src
+    post-install:
+      - install -Dm444 img/cutter.svg -t /app/share/icons/hicolor/scalable/apps/
+      - install -Dm444 ../${FLATPAK_ID}.appdata.xml -t /app/share/appdata
+    sources:
+      - type: archive
+        url: https://github.com/radareorg/cutter/archive/v1.4.tar.gz
+        sha256: f2fff960ea102fc8cf7d62fe25ee2c992b2e19857a4f6a2e308b3c9cba93e5b8
+      - type: file
+        path: org.radare.Cutter.appdata.xml
+
+  - jupyter.json
+    # evil magic to get the pip install sources (there has to be an easier way…):
+    # flatpak run --share=network --command=python3 org.kde.Sdk//5.11 -m pip install jupyter 2>/dev/null | sed -nur 's/^.*(http[^ ]*).*$/\1/p' | while read l; do ( mkdir dl; cd dl; wget -q "$l"; sha=$(sha256sum *| cut -d\  -f1); cd ..; mkdir -p .flatpak-builder/downloads/; mv dl .flatpak-builder/downloads/$sha; echo '{ "type": "file", "sha256": "'"$sha\", \"url\": \"$l\" }" ) || rm -rf dl; done
```

Comment:
The easier way is using pip generator from [here](https://github.com/flatpak/flatpak-builder-tools/tree/master/pip).